
  linuxthreads still uses these defines
  apply only for glibc 2.6 and above

--- config.make.in	2007-05-26 01:36:34.000000000 +0200
+++ config.make.in	2007-05-26 01:36:34.000000000 +0200
@@ -38,6 +38,11 @@
 all-warnings = @all_warnings@
 
 elf = @elf@
+have-protected = @libc_cv_asm_protected_directive@
+have-z-nodelete = @libc_cv_z_nodelete@
+have-z-nodlopen = @libc_cv_z_nodlopen@
+have-z-initfirst = @libc_cv_z_initfirst@
+have-z-relro = @libc_cv_z_relro@
 have-z-combreloc = @libc_cv_z_combreloc@
 have-z-execstack = @libc_cv_z_execstack@
 have-initfini = @libc_cv_have_initfini@
